store/actions/authenticateActions.js

```diff
@@ -0,0 +1,22 @@
+import commerce from '../../lib/commerce'
+import Router from 'next/router';
+
+import { SET_CUSTOMER } from './actionTypes';
+
+/**
+ * Async get customer
+ */
+export const setCustomer = () => (dispatch) => {
+  // First check is customer is logged in
+  const isLoggedIn = commerce.customer.isLoggedIn();
+  // If false redirect to /login
+  if (!isLoggedIn || isLoggedIn === false) {
+    Router.push('/login');
```

Comment:
```suggestion
    return;
```

We don't want to prevent new customers from using the store if they're not logged in 🙂 . We'll have to ensure the "about" page for a logged in customer does this redirect though.